src/assets/scripts/client/info/SimAirportInfoController.js

```diff
@@ -0,0 +1,189 @@
+import { leftPad } from '../utilities/generalUtilities';
+import { radiansToDegrees } from '../utilities/unitConverters';
+import { INVALID_NUMBER } from '../constants/globalConstants';
+
+/**
+ * Gets information about the current airport, specifically
+ * the airport's elevation, wind speed and direction, and altimeter.
+ *
+ * @class SimAirportInfoController
+ */
+export default class SimAirportInfoController {
+    /**
+     * @for SimAirportInfoController
+     * @constructor
+     */
+    constructor() {
+        /**
+         * @for SimAirportInfoController
+         * @property airport
+         * @type {AirportModel}
+         */
+        this.airport = null;
+
+        /**
+         * @for SimAirportInfoController
+         * @property wind
+         * @type {String}
+         */
+        this.wind = '';
+
+        /**
+         * @for SimAirportInfoController
+         * @property altimeter
+         * @type {Number}
+         */
+        this.altimeter = INVALID_NUMBER;
+
+        /**
+         * @for SimAirportInfoController
+         * @property elevation
+         * @type {String}
+         */
+        this.elevation = '';
+
+        /**
+         * @for SimAirportInfoController
+         * @property icao
+         * @type {String}
+         */
+        this.icao = '';
+
+        return this._init();
+    }
+
+    /**
+     * Binds this.update to this
+     *
+     * @for SimAirportInfoController
+     * @method _setupHandlers
+     * @private
+     */
+    _setupHandlers() {
+        this.update.bind(this);
+    }
+
+    /**
+     * Method to update information when airport is changed
+     *
+     * @for SimAirportInfoController
+     * @method update
+     * @param {AirportModel} airport
+     */
+    update(airport) {
+        this.airport = airport;
+
+        return this._recalculate();
+    }
+
+    /**
+     * @for SimAirportInfoController
+     * @method destroy
+     * @chainable
+     */
+    destroy() {
+        this.airport = null;
+        this.wind = null;
+        this.altimeter = null;
+        this.elevation = null;
+        this.icao = null;
+
+        return this;
+    }
+
+    /**
+     * @for SimAirportInfoController
+     * @method _init
+     * @private
+     */
+    _init() {
+        this._setupHandlers();
+
+        return this;
+    }
+
+    /**
+     * @for AirportGameInfoView
+     * @method _render
+     * @private
+     */
+    _recalculate() {
+        const airport = this.airport;
+
+        this.wind = this._buildWindAndGustReadout(airport.wind);
+        this.altimeter = this._generateAltimeterReading(airport.wind.speed);
+        this.elevation = airport.position[2];
```

Comment:
Is there a getter on the `AirportModel` that provides this value?  If not, one should be added so we can do `this.airportModel.elevation` here instead.

Reply:
done